models/news.js

```diff
@@ -3,7 +3,7 @@ const { Model } = require('sequelize');
 module.exports = (sequelize, DataTypes) => {
   class News extends Model {
     static associate(models) {
-      this.belongsTo(models.Category,{as: 'category'})
+      this.belongsTo(models.Category)
```

Comment:
El this no debería usarse en una función estática.

Reply:
ya cambie el this por el nombre del modelo.